src/stories/Library/Modals/modal-pause/ModalPause.tsx

```diff
@@ -18,59 +21,69 @@ export const ModalPause: React.FC<ModalPauseProps> = ({
-      <h3 className="text-header-h3">{title}</h3>
-      <div className="mt-48 color-secondary-gray">
-        <p className="text-body-medium-regular">{subtitle}</p>
-      </div>
+}) => {
+  const picker = useRef() as MutableRefObject<Instance>;
 
-      <div className="modal-pause__dropdowns mt-24">
-        <div className="datepickers">
-          <div className="datepicker">
-            <label htmlFor="startDate" className="text-body-medium-regular">
-              Startdato
-            </label>
-            <input
-              type="date"
-              name="startDate"
-              id="startDate"
-              className="datepicker-input"
-              aria-label="startDate"
-            />
-          </div>
+  const calendar = useCallback((node: Node | null) => {
+    const options: Partial<BaseOptions> = {
+      mode: "range",
+      defaultDate: ["2024-01-01", "2024-01-10"],
+      now: "2024-01-19",
+      altInput: true,
+      dateFormat: "d-m-Y",
+      static: true,
+      onReady: (selectedDates, dateStr, instance) => {
+        instance.altInput?.setAttribute("aria-label", "Pause period");
+      },
+    };
+
+    if (node !== null) {
+      picker.current = flatpickr(node, options);
+    }
+  }, []);
+
+  return (
+    <Modal shownModal={showModal} classNames="modal-padding modal-pause">
+      <div className="modal-pause__container">
+        <h3 className="text-header-h3">{title}</h3>
+        <div className="modal-pause__subtitle">
+          <p className="text-body-medium-regular">{subtitle}</p>
+        </div>
 
-          <div className="datepicker">
-            <label htmlFor="startDate" className="text-body-medium-regular">
-              Slutdato
-            </label>
-            <input
-              type="date"
-              name="endDate"
-              id="endDate"
-              className="datepicker-input"
-              aria-label="endDate"
-            />
+        <div className="modal-pause__date-range">
+          <div className="date-range">
+            <div className="date-range__input">
+              <label htmlFor="date-range" className="text-body-medium-regular">
+                Pause period
+              </label>
+              <input
+                id="date-range"
+                ref={calendar}
+                title="Hidden field"
+                aria-label="Hidden field"
+                type="text"
+              />
+            </div>
           </div>
         </div>
-      </div>
 
-      <div className="modal-pause__text-link mt-24 color-secondary-gray">
+        <div className="modal-pause__text-link">
+          <p className="text-body-small-regular">{textWithLink}</p>
+        </div>
         <p className="text-body-small-regular">
-          {textWithLink} <Links href="/" linkText={linkText} />
+          <Links href="/" linkText={linkText} />
         </p>
+        <div className="modal-pause__button mt-48">
+          <Button
+            buttonType="default"
```

Comment:
Since we're already adjusting this story, maybe it would be nice to adjust the button to keep it consistent. 
We normally use `buttonType="none"` if we don't want that arrow inside. And I thiiiiiink we don't use it in the final product?

Reply:
Check 👍